.git-blame-ignore-revs

```diff
@@ -0,0 +1,111 @@
+##
+# All formatting commit revs should be included in this file. All commits 
+# running `yarn fix` should only include the formatting changes. Any formatting 
+# config changes should be separated into two commits: the first commit changes 
+# formatting config, and the second runs the formater tool.
+#
+# Each entry should include a date, and a description of format config change 
+# where possible.
+#
+# Add to git config:
+#   git config blame.ignoreRevsFile .git-blame-ignore-revs
+# 
+# Verify all "yarn fix" commits are present in this file:
+#   git log --all -i --grep='yarn\sfix' --pretty=%H
+##
```

Comment:
Englilishs with betur grammers!

Also, I did line-wrapping at thought boundaries (commas or new sentences), which is a more oldschool technique:

```
# This file lists commits that perform bulk code-formatting operations.
# This allows `git blame` to skip over these commits and give better results.
#
# To use this file, set the following configuation:
#    git config blame.ignoreRevsFile .git-blame-ignore-revs
#
# Running `yarn fix` should always happen in its own commit,
# separate from any other changes, so git can skip those cleanly.
# Tooling changes, such as upgrading prettier or changing options,
# should happen in the commit right before running `yarn fix`.
#
# When adding commits to this file, always mention the date and
# any tooling changes from the previous commit.
```